src/schemas/__tests__/__snapshots__/Array.test.js.snap

```diff
@@ -11,6 +11,19 @@ Array [
 ]
 `;
 
+exports[`ArraySchema denormalization Class denormalizes a single entity 2`] = `
+Array [
+  Object {
+    "id": 1,
+    "name": "Milo",
+  },
```

Comment:
It seems that immutable data structures are converted to plain objects when serialized into snapshots, see: https://github.com/facebook/jest/issues/1622. In this case, the Array should be an actual array since the test uses `[1,2]` as input but each object is actually an `Immutable.Map` (which I manually verified is the case).